.github/workflows/diagram-cache.yml

```diff
@@ -0,0 +1,54 @@
+# Copyright DB Netz AG and contributors
+# SPDX-License-Identifier: Apache-2.0
+
+name: update_capella_diagram_cache
+
+on: push
+
+env:
+  CAPELLA_DOCKER_IMAGES_REVISION: main
+
+  # Can either be no or selected
```

Comment:
I'd rather point to the documentation. The available options may change in the future. 

https://dsd-dbs.github.io/capella-dockerimages/capella/introduction/#supported-dropins